walk-in-interview/src/main/webapp/log-in/applicant/script.js

```diff
@@ -0,0 +1,68 @@
+/**
+ * This file is specific to log-in/applicant/index.html. 
+ * It renders the fields on the page dynamically
+ */
+
+// TODO(issue/21): get the language from the browser
+const CurrentLocale = 'en';
+
+/**
+ * Import statements are static so its parameters cannot be dynamic.
+ * TODO(issue/22): figure out how to use dynamic imports
+ */
+import {AppStrings} from '../../strings.en.js';
+
+const COMMONG_STRINGS = AppStrings['log-in'];
+const STRINGS = AppStrings['applicant-log-in'];
+const HOMEPAGE_PATH = '../index.html';
```

Comment:
if this isn't meant to go to the homepage, then you should call it something else

Reply:
e.g., LOGIN_HOMEPAGE_PATH